packages/core/content-releases/server/src/controllers/release-action.ts

```diff
@@ -0,0 +1,36 @@
+import type Koa from 'koa';
+import { errors } from '@strapi/utils';
+import {
+  validateEntryContentType,
+  validateReleaseActionCreateSchema,
+  validateUniqueEntryInRelease,
+} from './validation/release-action';
+import { ReleaseActionCreateArgs, UserInfo } from '../types';
+
+const releaseActionController = {
+  async create(ctx: Koa.Context) {
+    const user: UserInfo = ctx.state.user;
+    const releaseActionArgs: ReleaseActionCreateArgs = ctx.request.body;
+
+    // Check permissions: releases can only be managed by super admins for now
+    if (!strapi.admin.services.role.hasSuperAdminRole(user)) {
+      throw new errors.ApplicationError('Content Releases is a superadmin only feature');
+    }
```

Comment:
This looks like something that should be handled in the routes file. @Convly Do you know if there is a policy in place to only allow super admins ? I can not find one

Reply:
The permissions will change and I will update this PR after this one: https://github.com/strapi/strapi/pull/18715 which will use policies